Tests/Functional/Utility/Lock/MySqlLockTestSecondInstance.php

```diff
@@ -61,13 +56,16 @@ protected function connect()
 
     public function test()
     {
-        if (!isset($_SERVER['argv']['1']) || !isset($_SERVER['argv']['2'])) {
+        if (!isset($_SERVER['argv']['1']) || !isset($_SERVER['argv']['2']) || !isset($_SERVER['argv']['3'])) {
             throw new \Exception('You have to specify the lock identifier and the testType', 1428853716);
```

Comment:
in der Fehlermeldung werden nur 2 Arguments erklärt, wir haben jetzt aber 3 Pflicht-Arguments